Formula/libosinfo.rb

```diff
@@ -14,13 +14,19 @@ class Libosinfo < Formula
   depends_on "gobject-introspection" => :build
   depends_on "intltool" => :build
   depends_on "pkg-config" => :build
+  depends_on "osinfo-db-tools" => :build
   depends_on "vala" => :optional
   depends_on "check"
   depends_on "gettext"
   depends_on "glib"
   depends_on "libsoup"
   depends_on "libxml2"
 
+  resource "osinfo-db" do
```

Comment:
Sorry, I meant it should be vendored into `osinfo-db-tools`. I don't understand why you're adding it to `libosinfo`

Reply:
Can you vendor `osinfo-db-tools` into `libosinfo` please?

Reply:
Done. IMO, that's even better than two separate packages because semantics of `--system` database is kept and user can still install `--local` database, separately.

`--system` database is shipped within libosinfo